_docs/concepts/what-is-istio/overview.md

```diff
@@ -1,29 +1,34 @@
 
-As monolithic applications transition towards a distributed microservice architecture they become more difficult to manage and understand. These 
-architectures need basic necessities such as discovery, load balancing, failure recovery, metrics and monitoring, and more complex operational requirements 
-such as A/B testing, canary releases, rate limiting, access control, and end-to-end authentication. The term service mesh is used to describe the network of
-microservices that make up applications and the interactions between them. As the service mesh grows in size and complexity, it becomes harder to understand
-and manage.
+Istio currently only supports service deployment on Kubernetes, though other environments will be supported in future versions.
 
-Istio provides a complete solution to satisfy these diverse requirements of microservice applications, by providing developers and operators with 
-behavioral insights and operational control over the service mesh as a whole. Istio does this by providing a number of key capabilities uniformly across the
+For detailed conceptual information about Istio components see our other [Concepts]({{home}}/docs/concepts/) guides.
+
+## Why use Istio?
+
+Istio addresses many of the challenges faced by developers and operators as monolithic applications transition towards a distributed microservice architecture. The term **service mesh** is often used to describe the network of
+microservices that make up such applications and the interactions between them. As a service mesh grows in size and complexity, it can become harder to understand
+and manage. Its requirements can include discovery, load balancing, failure recovery, metrics, and monitoring, and often more complex operational requirements 
+such as A/B testing, canary releases, rate limiting, access control, and end-to-end authentication.
+
+Istio provides a complete solution to satisfy the diverse requirements of microservice applications by providing
+behavioral insights and operational control over the service mesh as a whole. It provides a number of key capabilities uniformly across a
 network of services:
 
-- **Traffic Management**. Control the flow of traffic and API calls between services, make calls more reliable and make the network more robust in the face
+- **Traffic Management**. Control the flow of traffic and API calls between services, make calls more reliable, and make the network more robust in the face
 of adverse conditions.
  
-- **Observability**. Gain understanding of the dependencies between services, the nature and flow of traffic between them and be able to quickly identify 
+- **Observability**. Gain understanding of the dependencies between services, the nature and flow of traffic between them, and the ability to quickly identify 
```

Comment:
This doesn't sound right. It's saying:

Gain an understanding of: a) dependencies, b) nature and flow, c) the ability to quickly

That last component doesn't fit with the beginning of the sentence.